common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4607,6 +4614,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "PERSISTED_VIEW_READS_FROM_TEMPORARY_VIEW" : {
+    "message" : [
+      "Persisted view <persistedViewName> cannot reference temporary view <temporaryViewName> that will not be available outside the pipeline scope. Either make the persisted view temporary or persist the temporary view."
+    ],
+    "sqlState" : "42K0F"
+  },
```

Comment:
Error thrown when a persisted view try to read a temp view.

```sql
CREATE VIEW persisted_view AS SELECT * FROM temp_view
```

Creating a view over a temporary object is strictly disallowed in DBSQL and many other SQL products